litellm/__main__.py

```diff
@@ -0,0 +1,4 @@
+from .proxy.proxy_cli import run_server
+
+if __name__ == "__main__":
+    run_server()
```

Comment:
when is this file triggered?

Reply:
when using the package as a module, like using `python -m litellm` or `debugpy -m litellm`, basically allows the external debug to attach to the "litellm command"